docs/contributing.md

```diff
@@ -163,26 +163,24 @@ To use a preview build for a package within a project, you need to override the
 
 The [`create-release-branch`](https://github.com/MetaMask/create-release-branch) tool and [`action-publish-release`](https://github.com/MetaMask/action-publish-release) GitHub action are used to automate the release process.
 
-1. **Create a release branch.**
+1. **Initiate the Release Branch and Pre-Update Changelogs**
```

Comment:
It looks like for the other steps here, the first word is capitalized but the rest are not. I feel like that style is more friendly, so perhaps we want to adjust this to match?

Also, the tool is updating the changelogs, but the user does not need to do this; what they need to do is use the update changelogs to specify what they want to release. I like the word "initiate", because it implies more than just creating, but perhaps we want to adjust the summary a bit:

``` suggestion
1. **Initiate the release branch and specify packages to be released.**